R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -939,4 +939,67 @@ test_that("spark.randomForest Classification", {
   unlink(modelPath)
 })
 
+test_that("spark.gbt", {
+  # regression
+  data <- suppressWarnings(createDataFrame(longley))
+  model <- spark.gbt(data, Employed ~ ., "regression", maxDepth = 5, maxBins = 16, seed = 123)
+  predictions <- collect(predict(model, data))
+  expect_equal(predictions$prediction, c(60.323, 61.122, 60.171, 61.187,
+                                         63.221, 63.639, 64.989, 63.761,
+                                         66.019, 67.857, 68.169, 66.513,
+                                         68.655, 69.564, 69.331, 70.551),
+               tolerance = 1e-4)
+  stats <- summary(model)
+  expect_equal(stats$numTrees, 20)
```

Comment:
I see. Since there is no object represents ML Vector in SparkR currently, I'd like to convert the type of ```featureImportances``` from ```Vector``` to ```Array``` at ```GBTClassifierWrapper.scala```.
```
lazy val featureImportances: Array = gbtcModel.featureImportances.toArray
```
Users may sort or select on the feature importances, so return as an R Array or List should make more sense. Any thoughts?